src/version/client.go

```diff
@@ -8,16 +8,12 @@ import (
 	pb "github.com/pachyderm/pachyderm/v2/src/version/versionpb"
 )
 
-const (
-	// MajorVersion is the current major version for pachyderm.
-	MajorVersion = 2
-	// MinorVersion is the current minor version for pachyderm.
-	MinorVersion = 2
-	// MicroVersion is the patch number for pachyderm.
-	MicroVersion = 0
-)
-
 var (
+	// Overwritten at build time by linker
+	AppVersion = "2.2.0"
```

Comment:
Since this gets overwritten, the default value should probably be something noticeable (ie. `TEMPLATED` or something)

Reply:
good point. but won't folks building locally need a value? I will test this doesn't get overwritten when local builds are done. something like `TEMPLATED` will then result in a go int parse error.

Reply:
how about 0.0.0 or something obviously wrong?